packages/ckeditor5-emoji/ckeditor5-metadata.json

```
@@ -5,14 +5,21 @@
 			"className": "Emoji",
 			"description": "Allows for inserting emojis via a dropdown or by typing `:` followed by emoji name.",
 			"path": "src/emoji.js",
-			"docs": "features/emoji.html"
+			"docs": "features/emoji.html",
+			"requires": [
+				"EmojiMention",
+				"EmojiPicker"
+			]
```

```suggestion
			"requires": [
				"Mention"
			]
```

Metadata must include soft dependency requirements only, as the hard one (via class constructor) is available in run time (due to passing a reference instead of a string). Hence, I would go with the proposed change.